src/main/java/clickup/ui/BasePage.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package clickup.ui;
+
+import core.selenium.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * BasePage class, this class is the intermediary between the 'WebDriverManager' class and the pages.
+ *
+ * @author Maday Alcala
+ * @version 0.0.1
+ */
+public abstract class BasePage {
+    protected WebDriver driver;
+
```

Comment:
I think we can also add a WebDriverWait and a JavascriptExecutor attributes, so the subclasses can extend them and use if the situations to do so.